app.py

```diff
@@ -0,0 +1,103 @@
+from main.mains import Amity
+import argparse
+
+x = Amity()
+
+
+# Set up amity
+def initialise_amity():
+    x.read_file('main/input.txt')
+    x.offices
```

Comment:
**Code Quality: Critical**

This line of code is not used. It should not be here.